### ISSUE TITLE
[Research] OUI Compliance audit of the `maps_legacy` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4086]
Tracking issue for OUI compliance audit of the `maps_legacy` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
6.2K | [src/plugins/maps_legacy/public/map/_leaflet_overrides.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/maps_legacy/public/map/_leaflet_overrides.scss) | 
812 | [src/plugins/maps_legacy/public/map/_legend.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/maps_legacy/public/map/_legend.scss) | 
51 | [src/plugins/maps_legacy/public/map/index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/maps_legacy/public/map/index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
